ember-toucan-core/src/components/form/input-field.ts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+
+interface ToucanFormInputFieldComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    label?: string;
+    hint?: string;
+    error?: string;
+    isDisabled?: boolean;
+    readonly?: boolean;
+  };
+  Blocks: {
+    default: [];
+  };
+}
+
+// eslint-disable-next-line ember/no-empty-glimmer-component-classes
+export default class ToucanFormInputFieldComponent extends Component<ToucanFormInputFieldComponentSignature> {}
```

Comment:
If you're not using a constructor here, you could do this instead:

```js
import templateOnlyComponent from '@ember/component/template-only';

export default templateOnlyComponent<ToucanFormInputFieldComponentSignature>();
```

but if we do plan on adding a constructor eventually it's probably moot

Reply:
This should be a class now